runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -272,7 +273,11 @@ public void run() {
                                     ).collect(Collectors.toList())));
 
                     if (!entries.isEmpty()) {
-                        CorfuStreamEntries callbackResult = new CorfuStreamEntries(entries);
+                        Timestamp timestamp = Timestamp.newBuilder()
+                                .setSequence(logData.getGlobalAddress())
+                                .setEpoch(epoch)
+                                .build();
+                        CorfuStreamEntries callbackResult = new CorfuStreamEntries(entries, timestamp);
```

Comment:
Can the old streaming manager be removed ?

Reply:
There is a pending effort to remove all deprecated APIs and old code but that will come in another PR.